### PR TITLE
add information about transactions: #119

### PR DESCRIPTION
**Pull request by Guseyn**
add information about transactions: #119

### CODE REVIEW

**Comment by paulodamaso** [comment]
@g4s8 PLease assign me

**Comment by 0crat** [comment]
This pull request [#128](https://github.com/artipie/asto/issues/128) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/cfbbf3b4-9dae-48cb-91fb-ed47099a369e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/asto/google/GoogleStorage.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import com.artipie.asto.Transaction;
+import com.google.cloud.storage.StorageOptions;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Storage that holds data in Google storage.
+ *
+ * About transactions information can be found here:
```
Comment (carlosmiranda) [inline src/main/java/com/artipie/asto/google/GoogleStorage.java]: @Guseyn let's rephrase this a bit:

```
* Transactions are not supported. Atomicity is supported at the individual level.
* More information can be found here:
* https://stackoverflow.com/questions/23436244/google-cloud-storage-transactions
```

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@Guseyn ping

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Pull request [#128](https://github.com/artipie/asto/issues/128) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by 0crat** [comment]
Code review was too long (12 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by sereshqua** [comment]
@carlosmiranda please make sure you will at least 3 issues during next CR and that these would be about  design problems, not just cosmetic ones

**Comment by carlosmiranda** [comment]
@sereshqua confirmed

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request closed without merging**